test/fixtures/dialects/snowflake/snowflake_copy_into_location.sql

```diff
@@ -0,0 +1,7 @@
+COPY INTO '@public.cb_data_archive_prod/airflow-pipelines/2022-11-01/model/fct_rolling_active_users_l28/'
+            FROM "MODEL"."FCT_ROLLING_ACTIVE_USERS_L28"
+            FILE_FORMAT = (TYPE = PARQUET)
+            SINGLE = FALSE
+            MAX_FILE_SIZE = 1000000000
+            INCLUDE_QUERY_ID = TRUE
+			HEADER = TRUE
```

Comment:
Can we add a newline here:

```suggestion
			HEADER = TRUE

```

To avoid this red symbol that for some reason upsets me 😁

<img width="343" alt="image" src="https://user-images.githubusercontent.com/10931297/201280802-778d82cf-cce6-4740-b24f-3917d84c63fd.png">

FYI if your editor supports editorconfig then these are automatically added on save (plus another couple of similar things). VS Code needs a plugin for this but works well.